fpt/src/lr35902.rs

```diff
@@ -53,14 +45,20 @@ impl fmt::Debug for LR35902 {
     }
 }
 
+impl DebugInterface for LR35902 {
+    fn receive_command(&mut self, cmd: &DebugCmd) {
+        self.debugger.receive_command(cmd);
+    }
+}
+
```

Comment:
ooh, I see, this is delegating to `DebugCmd`, which happens to have a similar signature for now.